lib/super_digit.rb

```diff
@@ -1,15 +1,42 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity - O(n) if n is magnitude of number
+# Space Complexity - O(n)
 def super_digit(n)
```

Comment:
This works but it's O(d^2) where d is the number of digits because you do a summation O(d) inside of a recursive method (recurring ~ d times).